.github/workflows/prefect-installation-test.yml

```
@@ -0,0 +1,50 @@
+name: Prefect Installation Test
+
+on:
+  pull_request:
+  push:
+    paths:
+      - .github/workflows/prefect-installation-test.yml
+    branches:
+      - main
+      - prefect-install-test
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 6
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        python-version: [3.9]
+    name: ${{ matrix.os }} test installation
+    steps:
+      - name: Checking out git repo & Prefect submodule
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Setup conda
+        uses: s-weigand/setup-conda@v1
```

The default names for these are fine and what we use in our other repositories
```suggestion
      - uses: actions/checkout@v3
        with:
          submodules: true

      -  uses: s-weigand/setup-conda@v1
```